ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ipv6-prefix-lists.j2

```diff
@@ -1,10 +1,12 @@
 {# eos - ipv6-prefix-lists #}
-{% for ipv6_prefix_list in ipv6_prefix_lists | arista.avd.natural_sort %}
+{% for ipv6_prefix_list in ipv6_prefix_lists | arista.avd.convert_dicts('name') |
+arista.avd.natural_sort('name') %}
 !
-ipv6 prefix-list {{ ipv6_prefix_list }}
-{%     for sequence in ipv6_prefix_lists[ipv6_prefix_list].sequence_numbers | arista.avd.natural_sort %}
-{%         if ipv6_prefix_lists[ipv6_prefix_list].sequence_numbers[sequence].action is arista.avd.defined %}
-   seq {{ sequence }} {{ ipv6_prefix_lists[ipv6_prefix_list].sequence_numbers[sequence].action }}
+ipv6 prefix-list {{ ipv6_prefix_list.name }}
+{%     for sequence in ipv6_prefix_list.sequence_numbers | arista.avd.convert_dicts('sequence') |
+arista.avd.natural_sort('sequence') %}
```

Comment:
I think this wrapped to a new line by mistake. Same elsewhere in this and the other files.

Reply:
Yes, I have corrected this. Thanks